cosmic-core/engine/schema/src/main/java/com/cloud/dc/ClusterDetailsDaoImpl.java

```diff
@@ -91,10 +91,19 @@ public ClusterDetailsVO findDetail(final long clusterId, String name) {
         sc.setParameters("clusterId", clusterId);
         sc.setParameters("name", name);
 
-        final ClusterDetailsVO detail = findOneIncludingRemovedBy(sc);
+        ClusterDetailsVO detail = findOneIncludingRemovedBy(sc);
         if ("password".equals(name) && detail != null) {
             detail.setValue(DBEncryptionUtil.decrypt(detail.getValue()));
         }
+        if ("cpuOvercommitRatio".equals(name) || "memoryOvercommitRatio".equals(name)) {
+            final String detailDefaultValue = "1.0";
+            if (detail != null && detail.getValue() == null) {
```

Comment:
@eliasgomes Correct, but it really is null when there is no record in the db that could be retrieved. So, not in the db, but later in the code it is null and causing issues.